packages/react-urql/examples/1-getting-started/README.md

```diff
@@ -0,0 +1,26 @@
+## React
+
+This shows how to use URQL in the browser with React.
+
+A page displays a to-do list, and you can add items or mark them as complete.
+
+(Behind the scenes, we use a browser service worker to pretend to be a GraphQL server. This is done to avoid setting up a real GraphQL server for the example.)
+
+## Usage
+
```

Comment:
Something like this would be great for those not looking to pull the repo and build locally 

```suggestion
> Note: The quickest way to get started is to visit the live [codesandbox example](https://codesandbox.io/s/github/FormidableLabs/urql/master/packages/react-urql/examples/1-getting-started)
```

Reply:
nice idea, added, thanks! :)